assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -320,21 +474,23 @@ describe("DiversionPage", () => {
   })
 
   test("shows 'Regular Route' text when the detour is finished", async () => {
-    jest.mocked(fetchDetourDirections).mockResolvedValue(
-      Ok(
-        detourShapeFactory.build({
-          directions: [
-            { instruction: "Turn left on Main Street" },
-            { instruction: "Turn right on High Street" },
-            { instruction: "Turn sharp right on Broadway" },
-          ],
-        })
+    jest.mocked(fetchDetourDirections).mockReturnValue(
```

Comment:
Oh I should go through and change these back. I had a misconception about `mockResolvedValue`, which is why I changed these, but this is just diff noise at this point.

Reply:
[Feedback: Restore (correct) usages of mockResolvedValue](https://github.com/mbta/skate/pull/2580/commits/d34c0a0dfa11cdb8e586f6110f5e685e47fb701f)